alicloud/resource_alicloud_emr_cluster.go

```diff
@@ -0,0 +1,420 @@
+package alicloud
+
+import (
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/emr"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudEmrCluster() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudEmrClusterCreate,
+		Read:   resourceAlicloudEmrClusterRead,
+		Update: resourceAlicloudEmrClusterUpdate,
+		Delete: resourceAlicloudEmrClusterDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"cluster_type": {
```

Comment:
Do not support modified fields, please add forcenew, and the same in the document